build_debian.sh

```diff
@@ -318,6 +317,14 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     haveged                 \
     jq
 
+
+# Build ethtool package for generic platform
+if [[ $CONFIGURED_PLATFORM == generic ]]; then
+make -f slave.mk BLDENV=buster target/files/buster/ethtool
```

Comment:
This line should be indented four spaces.

Reply:
Fixed